demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2246,6 +2247,18 @@ export class DemoMeetingApp
     }
   }
 
+  createPauseResumeListener(tileState: VideoTileState): (event: Event) => Promise<void> {
+      return async (event: Event): Promise<void> => {
```

Comment:
Since the pause behavior is synchronous, so just want to know should this function execute synchronously?

Reply:
Oh hmm, don't know why I added the `async` modifier here.  Will remove.